src/components/ReanimatedSwipeable.tsx

```diff
@@ -262,28 +280,20 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       },
     });
 
-    const defaultProps = {
-      friction: 1,
-      overshootFriction: 1,
-    };
-
-    const {
-      friction = defaultProps.friction,
-      overshootFriction = defaultProps.overshootFriction,
-    } = props;
-
+    // %% remove or rename these, may cause 1. reloads, 2. name shadowing
     const overshootLeftProp = overshootLeft;
     const overshootRightProp = overshootRight;
 
-    const updateRightElementWidth = () => {
+    // %% should only run once, not every pan update
```

Comment:
What about those `%%` is this some convention that I'm not familiar with?

Reply:
I use `%%` to mark issues and questions i want to resolve before merging. 
I believe I removed all such comments before requesting your review, 
but as time passed I found new issues i have to address.

Reply:
Removed this comment in [c2ac345](https://github.com/software-mansion/react-native-gesture-handler/pull/3165/commits/c2ac3452729d20a36041a78b68860299472486da)